### PR TITLE
btr-web layout: header html

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/18130

*Description of changes:*
Tests to be done in a separate ticket

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: btr-web/btr-common-components/assets/styles/theme.scss [hunk 1]
```diff
@@ -117,5 +117,8 @@
 html,
 body {
   min-height: 100vh;
-  font-family: 'BCSans', Verdana, Arial, sans-serif;
+  font-family: 'BCSans', Verdana, Arial, sans-serif !important;
```
[the remark this answers is not in the document]
Comment (kialj876) [reply btr-web/btr-common-components/assets/styles/theme.scss:120]: oops no I will remove that. For some reason when it renders it looks slightly different than the other apps not sure why

File: btr-web/btr-layouts/components/bcros/Header.vue [hunk 2]
```diff
@@ -1,27 +1,107 @@
...
+const username = ref('Developer Test')
+const accountName = ref('Ministry of Citizen Services')
+
+const goToLogin = () => { isAuthenticated.value = true }
+const goToLogout = () => { isAuthenticated.value = false }
+const goToHome = () => {}
+const goToCreateAccount = () => {}
+
+const loginOptions = [
+  { label: 'BC Services Card', icon: 'i-mdi-account-card-details-outline', action: goToLogin },
```
[the remark this answers is not in the document]
Comment (kialj876) [reply btr-web/btr-layouts/components/bcros/Header.vue:67]: yes for sure -- I was going to add that in as part of the JS ticket but I can do it here

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-28-thhwvg04.web.app

File: btr-web/btr-layouts/components/bcros/Header.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (kialj876) [reply btr-web/btr-layouts/components/bcros/Header.vue]: I didn't add this for labels that are names (i.e. the account names) / short forms (i.e. IDIR)

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-28-thhwvg04.web.app

**Review by hfekete — APPROVED** [review]

**Pull request merged**